apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableExpectationSamples.kt

```diff
@@ -0,0 +1,241 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class IterableExpectationSamples {
+    @Test
+    fun toContainBuilder() {
+        expect(listOf("A", "B"))
+            .toContain
+            .inOrder // order specifier
+            .only
+            .values("A", "B")
+
+        fails {
+            expect(listOf(5, 3, 2, 2, 4))
+                .toContain
+                .inAnyOrder // order specifier
+                .atMost(2)
+                .entries({ // assertion group about the entries
+                    toBeGreaterThan(2)
+                })
+        }
+    }
+
+    @Test
+    fun notToContainBuilder() {
+        expect(listOf("A", "B"))
+            .notToContain
+            .elementsOf(setOf("C", "D"))
+
+        fails {
+            expect(listOf(1, 8, 5))
+                .notToContain
+                .entries({
+                    toBeGreaterThan(6)
+                })
+        }
+    }
+
+    @Test
+    fun toContainValues() {
+        expect(listOf("A", "B", "C")).toContain("B")
+
+        fails {
+            expect(setOf(4, 2, 1)).toContain(3)
+        }
+    }
+
+    @Test
+    fun toContainAssertion() {
+        val list = listOf(1, 2, 2, 4)
+        expect(list).toContain { toBeGreaterThan(3) }
+
+        fails {
+            expect(list).toContain { toBeGreaterThan(4) }
+        }
+    }
+
+    @Test
+    fun toContainAssertions() {
+        expect(listOf(1, 2, 2, 4))
+            .toContain(
+                { toBeLessThan(2) },
+                { toBeGreaterThan(3) }
+            )
+
+        fails {
+            expect(listOf(1, 2, 2, 4))
+                .toContain(
+                    { toEqual(3) }, // fails because no element in the list equals 3
+                    { toEqual(2) } // this assertion is not checked
```

Comment:
We generally try to align text under each other in samples. But that's not the main point, toContain checks all given entries, toEqual(2) holds, that's the reason why it is not reported. Thus change it to
```suggestion
                    { toEqual(3) }, // fails because no element in the list equals 3
                    { toEqual(5) } //  still evaluated and fails as well
```

Reply:
Good catch! I was trying to write two failing assertions and missed that.